content/blog/2018-11-13-antarctic.md

```diff
@@ -0,0 +1,347 @@
+
+Details of the files can be found in the `result` object, and those files could now be read with packages such as `raster`. However, we are collecting data from a range of sources, and so they will be different in terms of their grids, resolutions, projections, and variable-naming conventions, which tends to complicate these operations. In the next section we'll look at the `raadtools` package, which provides a set of tools for doing common operations on these types of data.
+
+#### Using those environmental data: raadtools
+
+[`raadtools`](https://github.com/AustralianAntarcticDivision/raadtools) is suite of functions that provide consistent access to a range of environmental and similar data, and tools for working with them. It is designed to work data collections maintained by the `bowerbird`/`blueant` packages.
+
+Load the package and tell it where our data collection has been stored:
+
+``` r
+library(raadtools)
+set_data_roots(my_data_dir)
+```
+
+Define our spatial region of interest and extract the bathymetry data from this region, using the ETOPO2 files we just downloaded:
+
+``` r
+roi <- round(c(range(ele$lon), range(ele$lat)) + c(-2, 2, -2, 2))
+bx <- readtopo("etopo2", xylim = roi)
+```
+
+And now we can make a simple plot of our our track superimposed on the bathymetry:
+
+``` r
+plot(bx)
+lines(ele$lon, ele$lat)
+```
+
+<img src="/img/blog-images/2018-11-13-antarctic/plotbathy-1.png" style="display: block; margin: auto;" />
+
+We can also extract the depth values along our track using the `extract()` function in `raadtools`:
+
+``` r
+ele$depth <- extract(readtopo, ele[, c("lon", "lat")], topo = "etopo2")
+```
+
+Plot the histogram of depth values, showing that most of the track points are located in relatively shallow waters:
+
+``` r
+ggplot(ele, aes(depth)) + geom_histogram(bins = 100) + theme_bw()
+```
+
+<img src="/img/blog-images/2018-11-13-antarctic/histbathy-1.png" style="display: block; margin: auto;" />
+
+This type of extraction will also work with time-varying data — for example, we can extract the sea-ice conditions along our track, based on each track point's location and time:
+
+``` r
+ele$ice <- extract(readice, ele[, c("lon", "lat", "date")])
+## points outside the ice grid will have missing ice values, so fill them with zeros
+ele$ice[is.na(ele$ice)] <- 0
+ggplot(ele, aes(date, ice, colour = lat)) + geom_path() + theme_bw()
+```
+
+<img src="/img/blog-images/2018-11-13-antarctic/temp2-1.png" style="display: block; margin: auto;" />
+
+Or a fancy animated plot, using `gganimate` (code not shown for brevity, but you can find it in the [page source](https://raw.githubusercontent.com/ropensci/roweb2/master/content/blog/2018-11-13-antarctic.md)). The hypnotic blue line shows the edge of the sea ice as it grows over the winter season:
```

Comment:
can you add text to say what the yellow/orange moving dot is?

Reply:
Done. The preview build is now failing but I think that's just because there's an empty topicid in the YAML